.github/workflows/run-tests.yml

```diff
@@ -203,6 +212,7 @@ jobs:
           allure_history: allure-history
 
       - name: Deploy report to Github Pages
+        timeout-minutes: 120
```

Comment:
2h to push git changes to GH?

Reply:
> 2h to push git changes to GH?

Yes, that can happen.
I did my worst-case testing by rebasing, deleting files, pushing with force and e.t.c
I also did it all on my personal runner with a slow internet connection.

Here is the slowest result:
https://github.com/vvarg229/neofs-node/actions/runs/4895100485/jobs/8740213411
`1h 6m 2s
`
I am surprised and shocked by this result, so I will try to rework the publication of the allure reports to a static web page as soon as possible.